lib/index.js

```diff
@@ -33,10 +36,11 @@ module.exports = function (ops, coverage) {
   }, function () {
     // Save refernce to this (bindless context cheat)
     var that = this;
+
     // Parse arguments
     var args = parseArgs(ops);
     // Using istanbul?
-    if (ist) {
+    if (ist && !outstream) {
```

Comment:
We need to figure out a way to support istanbul and outstream. Apparently only some of istanbul's output goes to `stderr`.